exercises/scrabble-score/spec/scrabble_score_spec.cr

```diff
@@ -0,0 +1,48 @@
+require "spec"
+require "../src/*"
+
+describe "ScrabbleScore" do
+  it "lowercase letter" do
+    ScrabbleScore.score("a").should eq(1)
+  end
+
+  it "uppercase letter" do
```

Comment:
should these be pending?

Reply:
Ah, yes. You are right. This is due to the test running not finishing as I was debugging the example. I'll have a fix for this soon.